commands/mute/tempmute.js

```diff
@@ -0,0 +1,127 @@
+                return msg.reply("You cannot temporarily mute a moderator or admin.");
+            } else if (toTempMute.roles.cache.some(role => role.name === "Muted")) {
+                return msg.reply("This user is already muted.");
+            } else {
+                if (reason === "") {
+                    return msg.reply("Please provide a reason for muting.");
+                }
+            }
+    
+            // Adds Muted role to user.
+            toTempMute.roles.add(msg.guild.roles.cache.find(role => role.name === "Muted"));
+            msg.channel.send(`${toTempMute} was muted by ${msg.member}.\nReason: ${reason}`);
+
+            // Sends user a DM notifying them of their muted status.
+            toTempMute.send("You've been muted for " + lengthOfTime + " for the following reason: ```" + reason + " ```");
+        }
+
+        // Outputs a message to the audit-logs channel
+        let channel = msg.guild.channels.cache.find(channel => channel.name === 'audit-logs')
+
+        const tempmuteEmbed = new Discord.MessageEmbed()
+            .setColor('#0099ff')
+            .setTitle(`${toTempMute.user.username}#${toTempMute.user.discriminator} was muted by ${msg.author.tag}:`)
+            .setDescription(reason)
+            .setThumbnail(`https://cdn.discordapp.com/avatars/${toTempMute.user.id}/${toTempMute.user.avatar}.png`)
+            .setTimestamp()
+            .setFooter(`${msg.guild.name}`);
+
+            channel.send(tempmuteEmbed);
+
+        const tempUnMuteEmbed = new Discord.MessageEmbed()
+            .setColor('#0099ff')
+            .setTitle(`${toTempMute.user.username}#${toTempMute.user.discriminator} was unmuted after ${lengthOfTime}:`)
+            .setThumbnail(`https://cdn.discordapp.com/avatars/${toTempMute.user.id}/${toTempMute.user.avatar}.png`)
+            .setTimestamp()
+            .setFooter(`${msg.guild.name}`);
+
+        // Add record to infractions table.
+        let now = new Date();
+        let timestamp = dateFormat(now, "yyyy-mm-dd HH:MM:ss");
+
+        var sqlInfractions = `INSERT INTO infractions (timestamp, user, action, length_of_time, reason, valid, moderator) 
+        VALUES ('${timestamp}', '${toTempMute}', 'cc!tempmute', '${lengthOfTime}', '${reason}', true, '${msg.author.tag}')`;
+
+        con.query(sqlInfractions, function (err, result) {
+            if (err) {
+            console.log(err);
+            } else {
+            console.log("1 record inserted into infractions.");
+            }
+        });
+
+        var sqlModLog = `INSERT INTO mod_log (timestamp, moderator, action, length_of_time, reason) 
+        VALUES ('${timestamp}', '${msg.author.tag}', '${msg}', '${lengthOfTime}', '${reason}')`;
+
+        con.query(sqlModLog, function (err, result) {
+            if (err) {
+            console.log(err);
+            } else {
+            console.log("1 record inserted into mod_log.");
+            }
+        });
```

Comment:
Same here, might as well put it all on one line instead of two different queries

Reply:
Done